src/Playwright.NUnit/PlaywrightTestAttribute.cs

```diff
@@ -130,6 +144,117 @@ private static string Indent(string text, int indent)
         {
             return text.Split(new[] { "\n" }, StringSplitOptions.None).Select(line => new String(' ', indent) + line).Aggregate((a, b) => a + "\n" + b);
         }
+
+        private (TestResult, bool) ExecuteWithTimeout(TestExecutionContext context)
+        {
+            var timeout = PlaywrightSettingsProvider.TestTimeout;
+            if (timeout == null || Debugger.IsAttached)
+            {
+                return (innerCommand.Execute(context), false);
+            }
+
+            if (CheckThatPlaywrightTimeoutIsSmallerThanNUnitTimeout(context, timeout.Value) is var result && result != null)
```

Comment:
nit: if you don't use `var` the is check implies not null. `is TestResult result` would automatically mean that `result` has value. 

https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEUCuA7NAExAGoAfAAQCYAGAWAChqBGRthxgFRgGcMAlXjgA2GAARQxAXjF4RwgNzsKzAJwAKSQEseY7n0E8R42EdEBKJRyZU9vAUNFiA3gF8gA==

Reply:
cool